### ISSUE TITLE
Allow Teams column class to be dynamic via config

### ISSUE THREAD

**Issue by eelco2k** [amendozaaguiar/filament-route-statistics#5]
it would be nice to select a different "teams class" column based on a teams_column_class config

```
TextColumn::make(class_exists('App\Models\Team') ? 'team.' . config('filament-route-statistics.team.column') : 'team_id')
                    ->label(__('filament-route-statistics::filament-route-statistics.table.columns.team'))
                    ->visible(class_exists('App\Models\Team') ? true : false)
                    ->searchable()
                    ->sortable(),
```

so instead of class_exists('App\Models\Team')  use something like: class_exists(config('filament-route-statistics.team.classname')) 

and in the filters:

```
if (class_exists(config('filament-route-statistics.team.classname'))) {
   $teamClass = config('filament-route-statistics.team.classname');
   return $teamClass::select('id', config('filament-route-statistics.team.column'))->pluck(config('filament-route-statistics.team.column'), 'id')->toArray();
                        }
```

**Comment by amendozaaguiar** [comment]
I am currently setting up an option that allows indicating from the configuration file whether it is handling a team and the column to display.

```

 'team' => [
        'model' => env('FILAMENT_ROUTE_STATISTICS_TEAM_MODEL', null),
        'column' => 'name'
    ],


```

**Comment by amendozaaguiar** [comment]
Resolved in commit [396580b](https://github.com/amendozaaguiar/filament-route-statistics/commit/396580bba81291b3552be6c291d5158bbbc0c552) [v2.3.0](https://github.com/amendozaaguiar/filament-route-statistics/releases/tag/2.3.0)
